client/my-sites/plan-features/index.jsx

```diff
@@ -288,6 +307,42 @@ class PlanFeatures extends Component {
 		} );
 	}
 
+	renderPopover() {
+		return (
+			<Popover
+				id="popover__plan-features"
+				isVisible={ this.state.showPopover }
+				context={ this.state.popoverReference }
+				position="right"
+				onClose={ this.closePopover }
+				className={ classNames(
+						'info-popover__tooltip',
+						'popover__plan-features'
+					) }
+				>
+					{ this.state.popoverDescripcion }
+			</Popover>
+		);
+	}
+
+	showPopover( el, popoverDescripcion ) {
+		const shouldHide = this.state.popoverReference === el &&
```

Comment:
Could we hold a reference to specific popover in `item` instead of `PlanFeatures`? Like in #7465 ? that would make it easier to merge #7465 :D but also be a bit cleaner, since it would create better code separation.

I am not a fan of exposing reference like this.